.github/workflows/test.yaml

```diff
@@ -48,13 +47,14 @@ jobs:
           # end to end functional test usage for non-distributed trial runs.
           - python-version: 3.6.7
             tox-env: nodeps-withcov-posix
-            trial-args: ''
+            trial-args: '-j 4'
```

Comment:
this is a drive-by fix, as parallel coverage should be enabled here.

It was accidentally disabled thinking that if left empty it will use the default.